anaconda_project/project.py

```diff
@@ -850,6 +851,11 @@ def _update_commands(self, problems, project_file, requirements):
         first_command_name = None
         commands = dict()
         commands_section = project_file.get_value('commands', None)
+
+        # from anaconda_project.internal.plugins import registry
+        plugins = plugins_api.get_all()
```

Comment:
It seems like this should be getting only plugins that define a command type, or somehow checking that